External/CoordGen/CMakeLists.txt

```diff
@@ -5,14 +5,14 @@ if(RDK_BUILD_COORDGEN_SUPPORT)
       set(MAEPARSER_DIR "${CMAKE_CURRENT_SOURCE_DIR}/maeparser")
     endif()
     if(NOT EXISTS "${MAEPARSER_DIR}/MaeParser.hpp")
-        set(SHA "83368293dcc0eb07562dadfb7728b8d18d23a6cb")
-        downloadAndCheckMD5("https://codeload.github.com/schrodinger/maeparser/tar.gz/${SHA}"
+        set(RELEASE_NO "1.0.0")
```

Comment:
Would it make sense to have the release nos configurable in the options?

Reply:
I don't think so. Someone who wants to have control over which version they use just needs to have a local copy of that version (then this download stuff doesn't happen). The idea of this is to have the automatic options pull a standard, well-tested version.